interface.rst

```diff
@@ -9,39 +9,46 @@ cable (but the screws to hold the connectors together will be mismatched).
+|   2 |  White        | 100kΩ to ground when disconnected         | Headphones Left         |
++-----+---------------+-------------------------------------------+-------------------------+
+|   3 |  Blue         | Unknown (Aux ground?)                     |                         |
++-----+---------------+-------------------------------------------+-------------------------+
+|   4 |  Black        | Aux Right                                 |                         |
++-----+---------------+-------------------------------------------+-------------------------+
+|   5 |  Black/White  | Aux Left                                  |                         |
++-----+---------------+-------------------------------------------+-------------------------+
+|   6 |  Green        | Ground                                    | Ground                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|   7 |  Yellow/White | Unused                                    | Unused                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|   8 |  Green/White  | Amplifier presence                        | Unknown signalling      |
++-----+---------------+-------------------------------------------+-------------------------+
+|   9 |  Gray         | Unused                                    | Unused                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|  10 |  Purple       | Unused                                    | Unused                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|  11 |  Brown        | Power supply                              | 3.3V                    |
++-----+---------------+-------------------------------------------+-------------------------+
+|  12 |  Yellow       | Rx                                        | Tx                      |
++-----+---------------+-------------------------------------------+-------------------------+
+|  13 |  Gray/White   | Tx                                        | Rx                      |
++-----+---------------+-------------------------------------------+-------------------------+
+|  14 |  Orange       | Unused                                    | Unused                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|  15 |  Red/White    | 0V if present                             | Console presence        |
++-----+---------------+-------------------------------------------+-------------------------+
+
+.. image:: images/console-port-diagram.jpg
+	:height: 156px
+	:alt: Console port pinout reference image.
+
+*Image reference of the port, showing which wire color is what on the original console cable.*
+**NOTE! This is from the sub/amp side, not the numbers on the cable!** *(yes the connector on the cable is labeled with numbers)*
```

Comment:
I'd just state that the connector is labelled with numbers that are different from those of the connector. It should be implied that these are the DB-15 pin numbers...